content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -69,21 +69,21 @@ different Kubernetes components.
 | `CSIMigration` | `true` | Beta | 1.17 | |
 | `CSIMigrationAWS` | `false` | Alpha | 1.14 | |
 | `CSIMigrationAWS` | `false` | Beta | 1.17 | |
-| `CSIMigrationAWSComplete` | `false` | Alpha | 1.17 | |
+| `CSIMigrationAWSComplete` | `false` | Alpha | 1.17 | 1.20 |
```

Comment:
Where are these `CSIMigration***` gates going? Are they deprecated or graduated?
We should move them to the second table if so.
Another question: shouldn't the second version number be "1.21"?

Reply:
They were “renamed” so they have been removed entirely. I think the last column is meant to be the last version they are available? They are all gone in 1.21.

Reply:
Up above the last column is "the last Kubernetes release in which you can still use a feature gate"

Reply:
Take `CSIMigrationAWSComplete` as an example, if they were effective in 1.20, but "deprecated" in 1.21. We should remove the lines from the first table and add the following lines to the second table.  The first table only maintains the gates that are still effective, aka, `alpha` or `beta`.

```suggestion
| `CSIMigrationAWSComplete` | `false` | Alpha | 1.17 | 1.20 |
| `CSIMigrationAWSComplete` | `false` | Deprecated | 1.21 | - |
```

Reply:
@tengqm in this case they are not simply deprecated they are removed entirely. Should I move them to the second table with no "Deprecated" line?